spec/controllers/admin/activities_controller_spec.rb

```diff
@@ -83,7 +83,7 @@
 
     it "generates a csv file" do
       get :index, format: :csv
-      expect(response.headers).to match(
+      expect(response.headers).to include(
```

Comment:
The response headers include some other unrelated ones. Nothing to worry about :)